fluentd/run.sh

```diff
@@ -105,5 +105,8 @@ else
     echo > $CFG_DIR/dynamic/es-ops-copy-config.conf
 fi
 
-
-exec fluentd $fluentdargs
+if [[ $DEBUG ]] ; then
+    exec fluentd $fluentdargs > /var/log/fluentd.log 2>&1
```

Comment:
i'd rather avoid needing to `eval` or make it look more complicated than necessary just to keep it a single line.

I was also confusing the `[[ $VERBOSE ]]` check with this one. I'm fine with the current implementation in this PR